File: app/src/main/java/com/example/alarm/SudokuPkg/game/Cell.java
package com.example.alarm.SudokuPkg.game;

public class Cell {
    public int row;
    public int col;
    public int value;
    public boolean isStarting = true;

    public Cell(int row, int col, int value) {
        this.row = row;
        this.col = col;
        this.value = value;
    }
}

File: app/src/main/java/com/example/alarm/sudoku/game/Cell.kt
package com.example.alarm.sudoku.game

class Cell(
    val row: Int,
    val col: Int,
    var value: Int,
    var isStartingCell: Boolean = false,
    var notes: MutableSet<Int> = mutableSetOf<Int>()
) {
    @JvmField
    var isStarting: Boolean = false
}